### DISCUSSION TITLE
Could you share 4xDRCT models in onnx format (FP32 and fp16)

### DISCUSSION THREAD

**Discussion by zelenooki** [Phhofm/models#2] [General]
Hi.
Chainner does not support drct(for now). Could you upload DRCT models in onnx format (Both fp16 and fp32)
fp32 cause I want to try model also in Hybrid, with vapoursynth vsmlrt module.
Thanks

**Comment by weter11** [comment]
Chainner 0.24 add support for it, but quality worse even ESRGAN models on my tests with 4xRealWebPhoto_v4_drct-l, strange.

**Reply by zelenooki** [reply]
because model are not for real world super resolution.
Author of DRCT said that real GAN should be released by the end of June.
